src/app/deleted-users/deleted-users/deleted-users.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-deleted-users',
+  templateUrl: './deleted-users.component.html',
+  styleUrls: ['./deleted-users.component.css']
+})
+export class DeletedUsersComponent implements OnInit {
+
+  route = "deleted-users";
```

Comment:
See my comment for active-users component html file above

Reply:
did this